src/api/app/views/layouts/webui2/_watchlist_dropdown.html.haml

```diff
@@ -17,7 +17,8 @@
         %small.text-muted
           Watch this project
     .dropdown-divider
-  - User.current.watched_project_names.each do |project|
-    = link_to project_show_path(project), class: 'dropdown-item' do
-      %span.icons-project
-      = raw(project.gsub(':', ':<wbr>'))
+  - User.current.watched_project_names.each_with_index do |project, index|
+    = link_to project_show_path(project), class: "dropdown-item text-wrap #{'border-top' unless index.zero?}" do
+      %i.fa.fa-cubes.fa.text-info.pt-2
+      %span.small
+        = raw(project.gsub(':', ':<wbr>'))
```

Comment:
We kind of have an helper for this `WebuiHelper.word_break` which also avoids `raw`, I can't remember but I think this was somehow considered more secure or something? @ChrisBr do you remember?